andela/prometheus-ah
# #159987748: User see the reading time for an article

#### What does this PR do?
- Allow readers see the reading time for articles

#### Description of Task to be completed?
- Update article model with a column for `readingTime`
- Create a migration file to add `readingTime` column to Article table
- Implement a controller for reading time at server/utils/ReadingTime.js
- Add reading time to articles when users create and update articles at server/controllers/ArticlesController.js
- Add assertion to test create article in articles.spec.js file

#### How should this be manually tested?
- clone this repository
- run npm install
- navigate to this branch by using git checkout ft-get-reading-time-159987748
- check .env.example to set up your .env file
- on your terminal type,npm run dev to start the server
- Make sure your Postgres database is running
- Launch postman, insert the authorization token in the header and make a POST request to localhost:port/api/articles to create an article with ``title, description, body`` request body

#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- [#159987748](https://www.pivotaltracker.com/story/show/159987748/comments/193981138)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-09 at 6 25 40 pm" src="https://user-images.githubusercontent.com/23406358/45267086-d434fa00-b45d-11e8-9600-dde83d091652.png">

#### Questions:
- None


[Finishes #159987748]
